### PR TITLE
Update metadata.yml

### PR DESCRIPTION
**Pull request by iamhyc**
update information for SIG deepin-surface.

### CODE REVIEW

**Review by xzl01 — APPROVED** [review]

File: sig/deepin-surface/metadata.yml [hunk 1]
```diff
@@ -2,15 +2,27 @@
...
   maintained:
+    - sig-deepin-surface
   package:
 members:
+  - handle: shiptux
+    id: 22735550
+  - handle: iamhyc
+    id: 9068301
+  - handle: xiekehui
+    id:
```
Comment (BLumia) [inline sig/deepin-surface/metadata.yml]:
```suggestion
    id: 20040331
```

**Review by BLumia — CHANGES_REQUESTED** [review]
See above

File: sig/deepin-surface/metadata.yml (hunk 1 above)
Comment (iamhyc) [reply sig/deepin-surface/metadata.yml]: Is it the same person on deepin forum?

Comment (BLumia) [reply sig/deepin-surface/metadata.yml]: If it isn't or if you are not sure about it, then it shouldn't be listed here.

If you would like to list a member that doesn't have GitHub account, you can edit MEMBERS.md to add their information manually.

Comment (iamhyc) [reply sig/deepin-surface/metadata.yml]: All the members have already been recorded in `MEMBERS.md` file.

If this metadata file is supposed to only record Github accounts, I can remove the entry for "xiekehui".

Comment (BLumia) [reply sig/deepin-surface/metadata.yml]: > If this metadata file is supposed to only record Github accounts, I can remove the entry for "xiekehui".

It is only for GitHub accounts currently. Please remove or comment out that item for now and maybe we can improve the yaml support in the future :)

**Review by BLumia — APPROVED** [review]

**Pull request merged**